addArray.java

```diff
@@ -0,0 +1,58 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.*;
+
+public class addArray {
+
+    public static void splitNum(List<Integer> c,int n)
+    {
+        if(n>0)
+        {
+            splitNum(c,n/10);
+            c.add(n%10);
+        }
+    }
+    public static void add(int[] a,int[] b,List<Integer> c,int alength,int blength,int a_current,int b_current)
```

Comment:
This is wrong. Your variable should describe its purpose. What does 'a' or 'b' describe? You could've had arr1 and arr2 or array1 and array2. Please change it again